eppoclient/client.go

```diff
@@ -2,26 +2,27 @@ package eppoclient
 
 import (
 	"fmt"
+	"time"
 )
 
 type Attributes map[string]interface{}
 
 // Client for eppo.cloud. Instance of this struct will be created on calling InitClient.
 // EppoClient will then immediately start polling experiments data from Eppo.
 type EppoClient struct {
-	configRequestor iConfigRequestor
-	poller          poller
-	logger          IAssignmentLogger
+	configurationStore *configurationStore
+	configRequestor    *configurationRequestor
+	poller             *poller
+	logger             IAssignmentLogger
 }
 
-func newEppoClient(configRequestor iConfigRequestor, poller *poller, assignmentLogger IAssignmentLogger) *EppoClient {
-	var ec = &EppoClient{}
-
-	ec.poller = *poller
-	ec.configRequestor = configRequestor
-	ec.logger = assignmentLogger
-
-	return ec
+func newEppoClient(configurationStore *configurationStore, configRequestor *configurationRequestor, poller *poller, assignmentLogger IAssignmentLogger) *EppoClient {
```

Comment:
ah so this is not a breaking change because it's used by [`initclient.go`](https://github.dev/Eppo-exp/golang-sdk/blob/main/eppoclient/eppoclient_e2e_test.go), which is the entry point for our API. 👌